foreman/data_refinery_foreman/surveyor/external_source.py

```diff
@@ -31,7 +31,7 @@ class PipelineEnums(Enum):
 
 class ProcessorPipeline(PipelineEnums):
     """Pipelines which perform some kind of processing on the data."""
-    MICRO_ARRAY_TO_PCL = "MICRO_ARRAY_TO_PCL"
+    AFFY_TO_PCL = "AFFY_TO_PCL"
```

Comment:
By the way, will more attributes be added here as we add more capabilities (such as ``AGILENT_TO_PCL`` or something like that)?